.github/workflows/build_and_deploy_dev.yml

```diff
@@ -26,6 +26,16 @@ jobs:
         with:
           node-version: '16'
           cache: 'npm'
+      - uses: actions/cache@v2
```

Comment:
trodde `actions/setup-node@v2` gjorde dette for npm? 
https://github.com/actions/setup-node#caching-packages-dependencies

Kanskje vi kun trenger nextjs-delen?

Reply:
Det er veldig mulig! Har nok bare copy-pastet denne inn fra en annen plass.

Ellers så har jeg kikket litt mer på hva cachingen av nextjs prøver å løse her. Sånn jeg har forstått det så vil den kun bruke cache hvis hverken package-lock.json eller noe av kildekoden har endret seg, så da vil det kanskje ikke ha så mye å si for hastigheten på byggene våre?

Reply:
spørs hvor ofte vi kjører "build-and-deploy" fra samme commit/tag? 
Vi kan jo teste det ut - og heller reverte hvis det ikke ser ut til å ha noen effekt? :)